docs/fsharp/language-reference/compiler-directives.md

```diff
@@ -52,6 +52,46 @@ let str = "Debugging!"
 #endif
 ```
 
+### Automatically Defined Symbols
+
+In addition to symbols explicitly defined via project settings or command-line options, **F# projects inherit a set of predefined symbols** based on the SDK, target framework, and build configuration.
+
+#### Debug, Tracing, and Nullable Settings  
+
+| Symbol    | Description |
+|-----------|-------------|
+| `DEBUG`   | Defined in debug builds when compiling with the `Debug` configuration. |
+| `TRACE`   | Defined when tracing is enabled in the project settings. |
+| `NULLABLE` | Defined when `<Nullable>enable</Nullable>` is set in the project file. |
+
+#### Target Framework Symbols  
```

Comment:
I am sorry for the misunderstanding.
Could those be instead solved by a link to the specific SDK documentation?

This is not being defined by F# and it does naturally evolve. If we keep a duplicate here, it would also require maintenace.

I think linking this very page https://learn.microsoft.com/en-us/dotnet/standard/frameworks#preprocessor-symbols would be best when it comes to completeness (see how many there are overall)